readme.md

```diff
@@ -2,9 +2,66 @@
 
 [![CircleCI](https://circleci.com/gh/HENNGE/aiodynamo.svg?style=svg)](https://circleci.com/gh/HENNGE/aiodynamo) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
+Asynchronous, pythonic DynamoDB client; **2x** faster than `aiobotocore`.
```

Comment:
it's also 2x faster than boto3/botocore fwiw.

Reply:
Hmm sounds like we need to test `aiobotocore` to be sure 🤔 
I'd assume they'd be equally slow or fast, but who knows?

Reply:
I ran the benchmarks against all three and the results were comparable. https://github.com/HENNGE/aiodynamo/pull/17#issuecomment-596409057

Reply:
tl;dr: boto3: 7.5s, botocore: 6.6s, aiobotocore: 6.6s, aiodynamo: 3.6s

Reply:
Confirmed, scanning 63K rows:
* `aiobotocore`: 19s, 23s, 20s
* `aiodynamo`: 9s, 9s